test/bin/manage_composer_config.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+#
+# A script for managing osbuild-composer.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=test/bin/common.sh
+source "${SCRIPTDIR}/common.sh"
+
+usage() {
+    cat - <<EOF
+${BASH_SOURCE[0]} (create [num_workers]|cleanup|create-workers <num_workers>)
+
+  -h           Show this help.
+
+create: Set up system for building images, start webserver.
+    [num_workers]: Number of workers to create for
+                   building in parallel.
+
+cleanup: Cancel any running builds, delete failed
+         and completed builds, and remove package 
+         sources other than the defaults.
+
+create-workers: Create multiple osbuild workers for 
+                building in parallel.
+    <num_workers>: Number of workers.
+
+EOF
+}
+
+action_create() {
+    "${ROOTDIR}/scripts/image-builder/configure.sh"
+
+    # Optionally create workers for building in parallel
+    if [ $# -ne 0 ]; then
```

Comment:
Let's remove this implicit configuration as we now have `create-workers` option?

Reply:
Removed